### PR TITLE
Examples for training, saving, loading, testing an agent with SB & RLlib

### LINKED ISSUE #14429: Load, play, rollout a trained/saved policy (without cli rollout.py)
I know this might be a duplicate, but there is still no clear section in the docs explaining how to do a simple rollout/render from a trained policy.
What I'm talking about is **code-based** (not command line based) equivalent of [stable baselines example like this](https://stable-baselines.readthedocs.io/en/master/guide/examples.html#basic-usage-training-saving-loading):
Screen Shot 2021-03-01 at 7 20 19 PM

In constrast, rllib provides only [this huge file that is really hard to understand](https://github.com/ray-project/ray/blob/master/rllib/rollout.py#L25), and can only be used via cli:
Screen Shot 2021-03-01 at 7 21 52 PM

It would be nice to have an example of how to access a trained policy itself, so that we can write a simple render loop like in stable-baselines above.

### PR DESCRIPTION
**Pull request by stefanbschneider**
## Why are these changes needed?

There have been a number of issues and questions regarding the transition from `stable_baselines` to Ray RLlib.
For newcomers, it can be a bit overwhelming given the wealth of different options that RLlib provides.
This intentionally simple example is supposed to help with the transition from SB to Rllib.

`sb2rllib_sb_example.py` shows how to train, save, load, and test a PPO agent on the CartPole environment with `stable_baselines` (inspired from the SB documentation).
`sb2rllib_rllib_example.py` shows a functionally equivalent script with RLlib. 

## Related issue number

Closes https://github.com/ray-project/ray/issues/14429

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by stefanbschneider** [comment]
@sven1977 Not sure if these examples are really useful and should be included in the list of examples? And whether to mention them somewhere in the docs? 
But I've seen different people struggling with the transition from SB to RLlib, which is why I thought a simple example may help here.

Is there any testing strategy for such examples? I just tested them locally.

**Comment by stefanbschneider** [comment]
Not sure why the Windows CI Tests are failing. It says "Timeout"

File: rllib/examples/sb2rllib_sb_example.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"""
+Example script on how to train, save, load, and test a stable baselines 2 agent
+Code taken and adjusted from SB2 docs:
+https://stable-baselines.readthedocs.io/en/master/guide/quickstart.html
+Equivalent script with RLlib: sb2rllib_sb_example.py
```
Comment (sven1977) [inline rllib/examples/sb2rllib_sb_example.py:5]:
```suggestion
Equivalent script with RLlib: sb2rllib_rllib_example.py
```

File: rllib/examples/sb2rllib_rllib_example.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+"""
+Example script on how to train, save, load, and test an RLlib agent.
+Equivalent script with stable baselines: sb2rllib_rllib_example.py
```
Comment (sven1977) [inline rllib/examples/sb2rllib_rllib_example.py:3]:
```suggestion
Equivalent script with stable baselines: sb2rllib_sb_example.py
```

**Review by sven1977 — APPROVED** [review]
Hey @stefanbschneider , this is super cool and helpful for a lot of users. Thanks so much! It'll also allow for direct comparison between the two, which is always good for catching issues.
I do think, we should mention these examples in the docs and we should add these to our tests (at least the rllib part), but we can do this in a follow up PR.
Maybe we should even have a new page for how-to-transition-from-library-xyz-to-rllib :) with these examples linked.

**Comment by stefanbschneider** [comment]
@sven1977 Great, I'm glad to hear that this is useful :) So is there anything that I should add/change for this PR?

How/where do you typically test these examples? Basically, the CI should be able to run the example script without errors, right? Or are there any real test cases/unit tests for these examples?

I agree, that would be nice to have a page in the docs that shows how to transition from framework XYZ to RLlib. I'm happy to help with the tests and/or the documentation for this.

**Comment by sven1977** [comment]
Thanks @stefanbschneider , we usually add these example scripts as tests to our ray/rllib/BUILD file (our bazel config, responsible for running all RLlib CI-tests). But this is ok, I think, we don't necessarily have to add this script, or we can do this in the follow-up PR, when we add more documentation.
Again, this is really great, thanks so much!

**Pull request merged**